papers/ice_nucleation_2024/calibration_setup.jl

```diff
@@ -247,8 +246,84 @@ function AIDA_IN05_IC(FT, data_file)
         eₛ = TD.saturation_vapor_pressure(tps, T₀, TD.Liquid())
         Sₗ = FT(e / eₛ)
     end
-    return [Sₗ, p₀, T₀, qᵥ, qₗ, qᵢ, Nₐ, Nₗ, Nᵢ, FT(0)]   #remove the last 2 elements, its J & r_l
+    return [Sₗ, p₀, T₀, qᵥ, qₗ, qᵢ, Nₐ, Nₗ, Nᵢ, FT(0)]
+end
+
+function AIDA_IN07_params(FT, w, t_max, t_profile, T_profile, P_profile, plot_name)
```

Comment:
Are you still using anywhere the generic `perf_model_params` and `perf_model_IC`? Or should we switch to only using the case specific ones?

Reply:
I'm using it as a unit test and there are also plots for it in the papers folder. I could get rid of the plots since we already know it is calibrate-able now?